src/components/walletconnect-list/WalletConnectLearnMoreButton.js

```diff
@@ -1,24 +1,22 @@
-import { debounce } from 'lodash';
-import React from 'react';
+import React, { useCallback } from 'react';
 import { Linking } from 'react-native';
 import Divider from '../Divider';
 import { ButtonPressAnimation } from '../animations';
 import { Centered, Row } from '../layout';
 import { Text } from '../text';
 import { padding } from '@rainbow-me/styles';
 
-const openWalletConnectWebsite = () =>
-  Linking.openURL('https://walletconnect.org/');
-
-const onPressLearnMore = () => debounce(openWalletConnectWebsite, 200);
-
 const WalletConnectLearnMoreButton = () => {
+  const openWalletConnectWebsite = useCallback(async () => {
```

Comment:
You can remove the async from here since you are not awaiting

Reply:
lol